packages/vue/src/examples/pages/thank-you/ThankYou.vue

```diff
@@ -135,26 +205,47 @@ export default {
   }
 }
 .feedback-button {
-  margin: var(--spacer) 0;
+  margin: var(--spacer-sm) 0;
 
   @include for-desktop {
-    margin: 0 var(--spacer-medium) 0 0;
+    --button-width: 25rem;
+    margin: var(--spacer-sm) 0 0 0;
   }
 }
 .heading {
-  --heading-padding: 3.75rem 0 0 0;
-
-  &--no-padding {
-    --heading-padding: 0;
-    --heading-text-align: left;
+  --heading-padding: var(--spacer-base) 0;
+  max-width: 11rem;
+  &--full {
+    --heading-padding: var(--spacer-base) 0;
+    max-width: 100%;
   }
-
   @include for-desktop {
-    --heading-text-align: left;
+    max-width: 100%;
   }
 }
 .paragraph {
-  font-size: var(--font-size-big);
-  line-height: 1.875rem;
+  font-family: var(--font-family-primary);
+  font-weight: var(--font-light);
+  color: var(--c-dark-variant);
+  margin: 0;
+  line-height: 1.75rem;
```

Comment:
```suggestion
  line-height: 1.6rem;
```
the best l-h for paragraphs 👍